### PR TITLE
Notify user of change to password, email and username

### LINKED ISSUE #814: Notify User on password change
### Epic

#393 
### Description

**As a:** user

**I want to:** be notified by email when my password is changed

**So that:** I can spot someone attempting to take over my account

### Which customers would this be availble to

- [X] All Users, (CE)
- [ ] Specific Project Types (EE)
- [ ] Specific Team Tiers (EE)
- [ ] Other (See comments)

### Acceptance Criteria

- [ ] criteria 1
- [ ] criteria 2
- [ ] criteria 3
- ...

### PR DESCRIPTION
**Pull request by Steve-Mcl**
closes #814, #815, #821

### CODE REVIEW

**Review by knolleary — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
<3 nicely split into commits @Steve-Mcl

File: forge/routes/api/shared/users.js [hunk 1]
```diff
@@ -112,7 +119,20 @@ module.exports = {
...
             await user.save()
 
-            // re-send verification email if a user was previously verifed and is now not verified
+            if (user.username !== originalUser.username) {
+                await app.postoffice.send(originalUser, 'UsernameChanged', {
+                    oldUsername: originalUser.username,
+                    newUsername: user.username
+                })
+            }
+            if (user.email !== originalUser.email) {
```
Comment (ZJvandeWeg) [inline forge/routes/api/shared/users.js:128]: @Steve-Mcl @knolleary The template in `forge/postoffice/templates/PasswordChanged.js` is never used?

Comment (ZJvandeWeg) [reply forge/routes/api/shared/users.js:128]: Ignore me, there's another file below.

Comment (knolleary) [reply forge/routes/api/shared/users.js:128]: It's used here: https://github.com/flowforge/flowforge/blob/main/forge/routes/api/user.js#L55

**Pull request merged**